tracer/build/_build/docker/centos7.dockerfile

```diff
@@ -66,6 +66,10 @@ RUN curl -sSL https://dot.net/v1/dotnet-install.sh --output dotnet-install.sh  \
 # Trigger first run experience by running arbitrary cmd
     && dotnet help
 
+# Install CppCheck
+RUN curl -sSL https://download-ib01.fedoraproject.org/pub/epel/7/x86_64/Packages/c/cppcheck-2.7-1.el7.x86_64.rpm --output cppcheck-2.7-1.el7.x86_64.rpm \
+    && sudo yum localinstall -y cppcheck-2.7-1.el7.x86_64.rpm
+
```

Comment:
Move this above the .net install + _potentially_ include in same ruby RUN to reduce layers

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3483/commits/87ff896d3f3fafb48844ebbbf0407d22bac7452a